### ISSUE TITLE
Staging Review finding: Focus management issues for file upload

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99262]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-7959f-2, Foreign Medical Program claim cover sheet
**Feature:** 

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/file-claim-form-10-7959f-2/upload-supporting-documents](https://staging.va.gov/health-care/foreign-medical-program/file-claim-form-10-7959f-2/upload-supporting-documents), when you upload a file, keyboard focus is sent to the top of the page. Keyboard users then have to tab back through the whole page to get back to where they started if they want to change that file, upload another file, or continue to the next step.

This is pretty disruptive to the keyboard experience. For screen reader users, keyboard focus helps determine what's announced next --- so when focus is sent to the top of the page, you hear something that sounds completely out of context and unrelated to what you were just doing, with no indication of whether adding that file worked.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Navigate through the form flow to Step 7.
2. Press [tab] to move focus to the "Upload file" button.
3. Press [enter] to activate the button.
4. Select a file.
5. Observe that keyboard focus is sent to the top of the page.
6. Press [tab] until focus reaches "Upload another file." (By my count it's 15 keystrokes.)

</details>

## Recommended action

In the Figma file shared at your Midpoint Review, it looked like you intended to use the [va-file-input component](https://design.va.gov/components/form/file-input). This component has been tested to handle focus management, and I would very much encourage you to use it.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by michaelclement** [comment]
Work is done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/33693). I investigated using the V3 file upload (multiple) component, but currently there is not a web-component version of the V3 File Upload Multiple component (for use in form config), so for the time being I opted to fix the focus issue as adding the v3 component will require more research.

Screenshot of focus after a file upload:

![Image](https://github.com/user-attachments/assets/72b21a26-96ec-40ed-9828-b111a64b006d)

**Comment by michaelclement** [comment]
@baltes this is ready for review. Thanks!
